### ISSUE TITLE
AttributeError: module 'py2neo' has no attribute 'cypher_escape'

### ISSUE THREAD

**Issue by jeffypie369** [stellargraph/stellargraph#1934]
### Describe the bug

*Receive the titled error when attempting to run the following line from the directed GraphSAGE demo*
    
    neo4j_sg = Neo4jStellarDiGraph(neo4j_graphdb, node_label="paper")

### To Reproduce

1. *Go to https://stellargraph.readthedocs.io/en/stable/demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.html*
2. *Under cell 5, the line of code that produces the error is*
`neo4j_sg = Neo4jStellarDiGraph(neo4j_graphdb, node_label="paper")`

3. *The error received is the title of this issue*

### Observed behavior

*AttributeError thrown as seen in the title*

### Expected behavior

*No errors thrown*

### Solution to this issue

1. *The issue is that in stellargraph/connector/neo4j/graph.py, the method raw_and_cypher calls py2neo.cypher_escape(raw). This is perhaps outdated as the py2neo docs has cypher_escape() under py2neo.cypher.*
2. *This bug is fixed if line 105 in graph.py is replaced as follows:*
Before:
`return raw, py2neo.cypher_escape(raw)`
After:
`return raw, py2neo.cypher.cypher_escape(raw)`

### Reference
https://py2neo.org/v4/cypher/index.html
![image](https://user-images.githubusercontent.com/39295219/120677287-8001bc00-c4c9-11eb-94cd-a3f8c775bf3d.png)

**Comment by Kashmira-ms** [comment]
neo4j_sg = Neo4jStellarGraph(neo4j_graphdb, node_label="paper")
neo4j_sg.cache_all_nodes_in_memory()


I'm also having the same issue while using Neo4jStellarGraph object

AttributeError                            Traceback (most recent call last)
<ipython-input-12-e46b81ea1339> in <module>()
----> 1 neo4j_sg = Neo4jStellarGraph(neo4j_graphdb, node_label="paper")
      2 neo4j_sg.cache_all_nodes_in_memory()

2 frames
/usr/local/lib/python3.7/dist-packages/stellargraph/connector/neo4j/graph.py in raw_and_cypher(raw)
    103         def raw_and_cypher(raw):
    104             if raw is not None:
--> 105                 return raw, py2neo.cypher_escape(raw)
    106             else:
    107                 return None, None

AttributeError: module 'py2neo' has no attribute 'cypher_escape'

Have you solved the issue

**Comment by jeffypie369** [comment]
Yep! If you do not wish to wait for StellarGraph to push an update, you can go to graph.py in your path
*/usr/local/lib/python3.7/dist-packages/stellargraph/connector/neo4j/graph.py*
and change line 105 according to my suggestion.

Remember to re-import the library after you have saved the file!

**Comment by Kashmira-ms** [comment]
AttributeError                            Traceback (most recent call last)
<ipython-input-34-e46b81ea1339> in <module>()
----> 1 neo4j_sg = Neo4jStellarGraph(neo4j_graphdb, node_label="paper")
      2 neo4j_sg.cache_all_nodes_in_memory()

2 frames
/usr/local/lib/python3.7/dist-packages/stellargraph/connector/neo4j/graph.py in raw_and_cypher(raw)
    103         def raw_and_cypher(raw):
    104             if raw is not None:
--> 105                 return raw, py2neo.cypher.cypher_escape(raw)
    106             else:
    107                 return None, None

AttributeError: module 'py2neo' has no attribute 'cypher_escape'


I do that and re import the libray again but still facing issue

**Comment by jeffypie369** [comment]
Hmm, try restarting the kernel?
